test/parallel/test-require-resolve.js

```diff
@@ -80,3 +80,13 @@ require(fixtures.path('resolve-paths', 'default', 'verify-paths.js'));
     assert.strictEqual(resolvedPaths.includes('/node_modules'), false);
   });
 }
+
+{
+  assert.strictEqual(require.resolve('https'), 'https');
+  assert.strictEqual(require.resolve('fs'), 'fs');
+
+  assert.throws(
+    () => require.resolve('unknown'),
```

Comment:
This needs to be passing when using the prefix, otherwise the original issue has not been fixed.
```suggestion
  assert.strictEqual(require.resolve('node:https'), 'node:https');
  assert.strictEqual(require.resolve('node:fs'), 'node:fs');

  assert.throws(
    () => require.resolve('node:unknown'),
```